synapse/models/biz.py

```diff
@@ -30,26 +30,26 @@ def getModelDefs(self):
                     'doc': 'A service which is performed by a specific organization.',
                 }),
                 ('biz:service:type:taxonomy', ('taxonomy', {}), {
-                    'doc': 'A taxonomy of service offering types.',
                     'interfaces': ('meta:taxonomy',),
+                    'doc': 'A hierarchical taxonomy of service types.',
                 }),
-                ('biz:dealstatus', ('taxonomy', {}), {
-                    'doc': 'A deal/rfp status taxonomy.',
+                ('biz:deal:status:taxonomy', ('taxonomy', {}), {
```

Comment:
one of  the few places we don't use 'type' (e.g., biz:deal:status:type:taxonomy). I can see :type being redundant or unnecessary in this particular case but wanted this to be a decision (vs oversight).

Note this was previously an enum (see L65) - though I can see enum being too constrained (and str being too loose) so taxonomy prolly makes sense.